lang/fr.json

```diff
@@ -0,0 +1,30 @@
+{
+	"_CodeMirror": {
+		"settings": {
+			"macroEditor": {
+				"name": "Utilisation pour l'édition de macros",
```

Comment:
"Utilisation" is the wrong word in this case. It shouldn't be passive I don't think, since that makes it a bit unclear. I would put "Utiliser pour l'édition de macros".